js-functions-arrays/task1.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Напишіть 3 види
+ *  функцій
+ * (function declaration, function expression, arrow function). 
+ * Кожна з них приймає 2 параметри, поверніть більше
+ *  значення. Викличте кожну функцію з 2 аргументами.
+ *  Результати виведіть у консоль.
+ */
+
+//function declaration
+function whatIsBiggerDeclaration(numA, numB) {
+    return (numA > numB) ? numA : numB;
+}
+console.log(whatIsBiggerDeclaration(7, 5))
+
+//function expression
+let whatIsBiggerExpression = function (numA, numB) {
+    return (numA > numB) ? numA : numB;
+}
+let result = whatIsBiggerExpression(7, 12);
+console.log(result)
+
+//arrow function
+const whatIsBiggerArrow = (numA, numB) => {
+    return (numA > numB) ? numA : numB;
+};
```

Comment:
```suggestion
const whatIsBiggerArrow = (numA, numB) => (numA > numB) ? numA : numB;
```

Reply:
You have only 1-row body. Return and "{}" are not necessary to use, ifyou have only one code line.